docs/_docs/focus/convert.md

```diff
@@ -57,35 +62,33 @@ The following mapping is assuming you have all amortized cost rows and only comm
+| InvoiceIssuerName          | PartnerName                                         | If PartnerName is empty, use `Microsoft`                                                                                                                              |
 | ListCost                   | EA: Not available<br>MCA: PaygCostInBillingCurrency | None                                                                                                                                                                  |
 | ListUnitPrice              | EA: PayGPrice<br>MCA: PayGPrice \* ExchangeRate     | None                                                                                                                                                                  |
 | PricingCategory            | PricingModel                                        | If "OnDemand", then `Standard`; if "Spot", then `Dynamic`; if "Reservation" or "Savings Plan", then `Committed`; otherwise, null                                      |
-| PricingQuantity            | Quantity                                            | Map UnitOfMeasure using [Pricing units data file](../../_reporting/data/README.md#-pricing-units) and divide Quantity by the PricingBlockSize<sup>2</sup>             |
-| PricingUnit                | UnitOfMeasure                                       | Map using [Pricing units data file](../../_reporting/data/README.md#-pricing-units)                                                                                   |
+| PricingQuantity            | Quantity / x_PricingBlockSize (lookup)              | Map UnitOfMeasure to x_PricingBlockSize using [Pricing units data file](../../_reporting/data/README.md#-pricing-units)<sup>1</sup>                                   |
+| PricingUnit                | DistinctUnits (lookup)                              | Map UnitOfMeasure to DistinctUnits using [Pricing units data file](../../_reporting/data/README.md#-pricing-units)                                                    |
 | ProviderName               | `Microsoft`                                         | None                                                                                                                                                                  |
 | PublisherName              | PublisherName                                       | None                                                                                                                                                                  |
 | RegionId                   | focus:RegionName                                    | Lowercase and remove spaces                                                                                                                                           |
```

Comment:
We could. The reason I used the `RegionName` reference is because doing another lookup takes more time. This is just faster and aligns with the general guidance we have from the ARM team about how to handle location IDs.

Reply:
How does one obtain `focus:RegionName`?  It's not a column.  Confused on that.

Reply:
`RegionName` is the next column